qtgui/SourceImage.cpp

```diff
@@ -28,30 +28,30 @@ SourceImage::SourceImage(QQuickItem* parent)
 , newBottomRight{ -1, -1 }
 , abort{ false }
 {
-  logging::LogStream::instance().setLogLevel(logging::Level::INFO);
+  logging::LogStream::instance().setLogLevel(logging::Level::NOTE);
```

Comment:
could this also be shortened by creating a function in logging module that returns LogStream::instance()?

Reply:
introduced logging::logger() to return LogStream::instance().